example/src/styles/global.css

```
@@ -22,10 +25,19 @@
 
   html,
   body {
-    @apply bg-[#080808] text-white;
+    @apply bg-bg text-fg;
     -webkit-tap-highlight-color: transparent;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)